### PR TITLE
[Template] Remove Outdated Stress Assets

### PR DESCRIPTION
**Pull request by jsquire**
# Summary

The focus of these changes is to update the template to remove references to the outdated stress framework that should no longer be used and replace with a pointer to the onboarding materials for the current stress framework.

### CODE REVIEW

File: sdk/template/.content/stress/README.md [hunk 1]
```diff
@@ -0,0 +1 @@
+For information on reliability testing with the Azure SDK stress test framework, please refer to the [Reliability testing wiki](https://dev.azure.com/azure-sdk/internal/_wiki/wikis/internal.wiki/463/Reliability-Testing).
```
Comment (richardpark-msft) [inline sdk/template/.content/stress/README.md:1]:
```suggestion
For information on reliability testing with the Azure SDK stress test framework, please refer to the [Reliability testing wiki](https://aka.ms/azsdk/stress).
```

Comment (richardpark-msft) [reply sdk/template/.content/stress/README.md:1]: Looks like this shows up in a few other spots too?

**Review by richardpark-msft — CHANGES_REQUESTED** [review]
Minor, but can you use the short link in case we decide to move things around?

File: sdk/template/Azure.Template/stress/README.md [hunk 2]
```diff
@@ -0,0 +1 @@
+For information on reliability testing with the Azure SDK stress test framework, please refer to the [Reliability testing wiki](https://dev.azure.com/azure-sdk/internal/_wiki/wikis/internal.wiki/463/Reliability-Testing).
```
Comment (jsquire) [inline sdk/template/Azure.Template/stress/README.md:1]:
```suggestion
For information on reliability testing with the Azure SDK stress test framework, please refer to the [Reliability testing wiki](https://aka.ms/azsdk/stress).
```

**Review by richardpark-msft — APPROVED** [review]
Looks great!

**Review by heaths — APPROVED** [review]

**Pull request merged**